components/script/dom/macros.rs

```diff
@@ -495,6 +495,13 @@ macro_rules! window_event_handlers(
         event_handler!(unhandledrejection, GetOnunhandledrejection,
                        SetOnunhandledrejection);
         event_handler!(unload, GetOnunload, SetOnunload);
+        event_handler!(vrdisplayconnect, GetOnvrdisplayconnect, SetOnvrdisplayconnect);
```

Comment:
Yes, I'll remove these ones. I started implementing 1.1 API but moved to 1.2. These events are already moved to VRDisplay instances (in the implementation and samples)